web/src/app/tenants/hooks.ts

```diff
@@ -1,44 +1,71 @@
 import { FormikHelpers } from "formik";
-import { SearchRoom } from "../rooms/hooks";
+import { RoomsFormValues, SearchRoom } from "../rooms/hooks";
 import { ActionButtonProps, Column, TableActions } from "@/components/Table";
-import { useCallback, useState } from "react";
-import { SelectFieldProps } from "@/components/Select";
+import { useCallback, useEffect, useState } from "react";
+import { OptionSelect, SelectFieldProps } from "@/components/Select";
 import { InputFieldProps, Field } from "@/components/hooks/useModal";
 
-interface TenantFormValues {
+interface RoomHistoryValues {
+  id: number;
+  roomId: number;
+  userId: number;
+}
+
+/**
+ * TenantFormValues properties
+ * extends of {@link RoomHistoryValues}  pick only property roomId
+ * extends of {@link RoomsFormValues} pick only property amount
+ */
+interface TenantFormValues
+  extends Partial<Pick<RoomHistoryValues, "roomId">>,
+    Partial<Pick<RoomsFormValues, "amount">> {
   id?: number;
   name: string;
   contact: string;
   address: string;
 }
 
+const inititialFormValues: TenantFormValues = {
+  name: "",
+  contact: "",
+  address: "",
+  roomId: undefined,
+  amount: undefined,
```

Comment:
pwede nani wala ang `roomId` ug ang `amount`, ang `roomId` kay undefined, and `amount` kay dili mana iapil ug send sa API kay for display raman na